bootstrap/manifest_template.json

```diff
@@ -131,6 +131,34 @@
     "data": "{{ getenv \"CONTROLLER_KEY\" }}"
   },
   {
+    "id": "redis-api",
+    "app": {
+      "name": "redis-api",
+      "meta": {"flynn-system-app": "true"}
+    },
+    "action": "run-app",
+    "release": {
+      "env": {
+        "FLYNN_REDIS": "redis",
+        "CONTROLLER_KEY": "{{ (index .StepData \"controller-key\").Data }}",
+        "REDIS_IMAGE_URI": "$image_repository?name=flynn/redis&id=$image_id[redis]"
+      },
+      "processes": {
+        "web": {
+          "ports": [{"port": 80, "proto": "tcp"}],
+          "cmd": ["api"]
+        }
+      }
+    },
+    "artifact": {
+      "type": "docker",
+      "uri": "$image_repository?name=flynn/redis&id=$image_id[redis]"
+    },
+    "processes": {
+      "web": 2
+    }
+  },
+  {
```

Comment:
If you don't follow `run-app` with `add-app` the application won't be added to the `controller`. You should probably use `deploy-app` instead though and move this until after the controller is fully setup as it's not required to get the controller going.